lib/form.js

```diff
@@ -33,7 +33,7 @@ module.exports = function(req, opts){
       if (err) return done(err);
 
       try {
-        done(null, qs.parse(str));
+        done(null, qs.parse(str, opts.queryString));
```

Comment:
I thought of it, but ended up with the full `queryString` as it is more verbose. "`qs`" is fine for a module name, but we're dealing user code at this point and there's nothing wrong with having a FQN for this option. IMO.